SPECS-EXTENDED/ocaml-gettext/ocaml-gettext.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "ocaml-gettext-0.3.8-3aecf8e5350f.tar.gz": "007dc792d564257f10a4f62695f97547c9ffe22f6c3aa01c7d5747a84c3fc369"
+  "gettext-v0.4.2.tbz": "8b672c7c521b8ac753c6a90925243cdd367dd5202e7c1e5d1a2507b11ad5d6a7"
```

Comment:
I think you've checked-in this binary by mistake.

Reply:
Fixed, and added `*.tbz` files to our `.gitignore`